packages/expo-camera/ios/VideoOptions.swift

```diff
@@ -0,0 +1,57 @@
+import ExpoModulesCore
+
+struct CameraRecordingOptions: Record {
```

Comment:
Since the main struct returned from this file is `CameraRecordingOptions`, I think the filename should be `CameraRecordingOptions.swift` 😉